x/storage/types/tx.pb.go

```diff
@@ -3150,7 +3150,7 @@ var fileDescriptor_ddb71b028305a3cc = []byte{
 	0x40, 0xd1, 0x21, 0xa6, 0xa5, 0x99, 0x0e, 0xea, 0x89, 0x7c, 0x53, 0x97, 0x35, 0x1f, 0xf6, 0xf4,
 	0x36, 0xf6, 0x7c, 0x9c, 0x1a, 0x2e, 0x61, 0xd5, 0x3f, 0xc1, 0xd4, 0x1e, 0x10, 0xcd, 0x17, 0x2a,
 	0xf8, 0x56, 0xbe, 0x0b, 0x40, 0x5d, 0xe4, 0x62, 0x66, 0x6a, 0x3f, 0x0b, 0xa7, 0x11, 0xdf, 0xf7,
-	0x81, 0xaa, 0xb4, 0x46, 0x69, 0xc7, 0x73, 0xe2, 0x6c, 0x66, 0x4e, 0x9c, 0x7b, 0xf2, 0x6c, 0x2d,
+	0x81, 0xaa, 0xb4, 0x46, 0x69, 0xc7, 0x73, 0xe2, 0x74, 0x66, 0x4e, 0x9c, 0x7b, 0xf2, 0x6c, 0x2d,
```

Comment:
Isn't this a hardfork to modify this content?

Reply:
reverted